### PR TITLE
address most include/remove notes in #32

### PR DESCRIPTION
**Pull request by phoddie**
This PR tries to address all of the include/remove notes except for editing the includes in the platform section (that looks possible but will take some more time).

### CODE REVIEW

**Comment by phoddie** [comment]
I've resolved the linter checks that I understand. The remaining issues get into TypeScript details that I'm not certain how to resolve. @HipsterBrown, can you help?

File: src/commands/include.ts [hunk 1]
```diff
@@ -30,22 +30,32 @@ const command: GluegunCommand = {
...
+    const modulePath = `$(MODDABLE)/modules/${String(moduleName)}/manifest.json`
     await patching.update(manifestPath, (manifest) => {
-      manifest.include.push(
-        `$(MODDABLE)/modules/${String(moduleName)}/manifest.json`
-      )
+      if (!("include" in manifest))
+        manifest.include = []
+      if (typeof manifest.include === "string")
+        manifest.include = [manifest.include]
+      if (!manifest.include.includes(modulePath)) {
```
Comment (HipsterBrown) [inline src/commands/include.ts:52]:
```suggestion
      if (manifest.include.includes(modulePath) === false) {
```

File: src/commands/include.ts [hunk 2]
```diff
@@ -30,22 +30,32 @@ const command: GluegunCommand = {
         modules !== undefined
           ? modules.map((mod) => collectChoicesFromTree(mod)).flat()
           : []
+      const filtered = moduleName ? choices.filter((mod: string) => mod.includes(String(moduleName))) : choices
```
Comment (HipsterBrown) [inline src/commands/include.ts:33]:
```suggestion
      const filtered = moduleName !== undefined ? choices.filter((mod: string) => mod.includes(String(moduleName))) : choices
```

File: src/commands/include.ts [hunk 3]
```diff
@@ -30,22 +30,32 @@ const command: GluegunCommand = {
...
+        manifest.include = []
+      if (typeof manifest.include === "string")
+        manifest.include = [manifest.include]
+      if (!manifest.include.includes(modulePath)) {
+        manifest.include.push(
+          modulePath
+        )
+      }
+      if (manifest.include.length === 1)
+        manifest.include = manifest.include[0]
```
[the remark this answers is not in the document]
Comment (phoddie) [reply src/commands/include.ts:58]: That's correct. I made changes in both `include` and `remove` to accommodate that.

File: src/commands/include.ts [hunk 4]
```diff
@@ -30,22 +30,32 @@ const command: GluegunCommand = {
...
           ? modules.map((mod) => collectChoicesFromTree(mod)).flat()
           : []
+      const filtered = moduleName ? choices.filter((mod: string) => mod.includes(String(moduleName))) : choices
       const { mod: selectedModule } = await prompt.ask([
         {
           type: 'autocomplete',
           name: 'mod',
           message: 'Here are the available modules:',
-          choices,
+          choices: filtered.length ? filtered : choices,
```
[the remark this answers is not in the document]
Comment (phoddie) [reply src/commands/include.ts]: Thanks for the tips. (Suppressing all comments on the linter!)

**Review by HipsterBrown — APPROVED** [review]
🎉

**Pull request merged**